coalib/bearlib/languages/documentation/DocstyleDefinition.py

```diff
@@ -152,9 +168,29 @@ def load(cls, language: str, docstyle: str, coalang_dir=None):
             raise KeyError("Language {!r} is not defined for docstyle {!r}."
                            .format(language, docstyle))
 
+        def get_from_docstyle_settings(*args):
+            values = list()
+            for req_setting in args:
+                setting = dict(
+                    docstyle_settings.contents.items()).get(req_setting)
+                try:
+                    setting = setting.value
+                except AttributeError:
+                    setting = ""
+                values.append(setting)
+            return values
+
+        values = get_from_docstyle_settings('param_start', 'param_end',
+                                            'return_sep')
+
+        metadata = cls.Metadata._make(values)
+
+        ignored_settings = ("comment", "param_start", "param_end", "return")
```

Comment:
you should make out of `param_start`, `param_end` and `return` a separate tuple, which you can use above for
`get_from_docstyle_settings(*special_values)` and here you concat them with `"comment"` :

```
ignored_settings = ("comment",) + special_settings
```

`special_settings` is maybe a bad name, but I hope you get the principle :D